yarpcconfig/chooser_test.go

```diff
@@ -363,6 +364,25 @@ func TestChooserConfigurator(t *testing.T) {
 				_ = list
 			},
 		},
+		{
+			desc: "use fewest-pending-requests chooser",
+			given: whitespace.Expand(`
+				outbounds:
+					their-service:
+						unary:
+							fake-transport:
+								fewest-pending-requests:
+									fake-updater: {}
+			`),
+			test: func(t *testing.T, c yarpc.Config) {
+				outbound := c.Outbounds["their-service"]
+				unary := outbound.Unary.(*yarpctest.FakeOutbound)
+				chooser := unary.Chooser().(*peer.BoundChooser)
+				list, ok := chooser.ChooserList().(*pendingheap.List)
+				require.True(t, ok, "use pending heap")
+				_ = list
```

Comment:
Question: why this anonymous assignment at last line? It seems useless. But I see lots of similar usage in test, maybe add a comment could be helpful.

Reply:
good catch! bad copy paste / me being lazy :/